### PR TITLE
build(cargo): bump up swc_core

### PR DESCRIPTION
**Pull request by kwonoj**
This PR bumps up swc_core to close-to-latest to pick up some fixes.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,7 +18,7 @@ serializable = ["serde"]
 [dependencies]
 markdown = "1.0.0-alpha.4"
 serde = { version = "1", optional = true }
-swc_core = { version = "0.44.0", features = [
+swc_core = { version = "0.43.23", features = [
```
Comment (ChristianMurphy) [inline Cargo.toml:21]: I'm a bit confused.
To get fixes, we need to go from `0.44` back to `0.43`?

Comment (kwonoj) [reply Cargo.toml:21]: oh wait, something is not right 🤔

Comment (kwonoj) [reply Cargo.toml:21]: Yes I think you're right, there's something weird happening on my end while working on bump. Thanks for catching.

**Comment by wooorm** [comment]
btw, as I understand, `0.44.0` is treated on crates as `^0.44.0` would on npm. I’d prefer to reduce the noise by sticking with the only the required, significant change!

**Comment by kwonoj** [comment]
> as I understand, 0.44.0 is treated on crates as ^0.44.0 would on npm. I’d prefer to reduce the noise by sticking with the only the required, significant change!

swc_core does not strictly follows semver and there are the cases we need explicit bump time to time unfortunately. Reason I hope to get some automatic update / publish.

**Comment by wooorm** [comment]
Semver is still semver. And crates.io follows that.
If SWC doesn’t follow that, stuff will break everywhere anyway.

It’s not feasible to create a release of this package every day, and every other package that depends on SWC.

If possible, I think it’s better to follow semver the way crates.io supports it.

**Pull request closed without merging**